homeassistant/components/emulated_kasa/__init__.py

```diff
@@ -0,0 +1,105 @@
+"""Support for local power state reporting of entities by emulating TP-Link Kasa smart plugs."""
+import logging
+from time import time
+
+from sense_energy import PlugInstance, SenseLink
```

Comment:
Its a bit odd that it lives in the `sense_energy` package, but it could be broken out later.